src/readii/loaders.py

```
@@ -14,7 +14,9 @@
 import pydicom
 import SimpleITK as sitk
 from imgtools import io
-from imgtools.ops import StructureSetToSegmentation
+from imgtools.io.readers import read_dicom_auto
+# from imgtools.ops import StructureSetToSegmentation
+from imgtools.coretypes import RTStructureSet
```

Ruff (0.8.2)

`imgtools.coretypes.RTStructureSet` imported but unused

Remove unused import: `imgtools.coretypes.RTStructureSet`

(F401)